packages/camera/camera_windows/windows/capture_controller.cpp

```diff
@@ -578,6 +578,7 @@ void CaptureControllerImpl::StartPreview() {
   texture_handler_->UpdateTextureSize(preview_frame_width_,
                                       preview_frame_height_);
 
+  // TODO(loic-sharma): This does not handle duplicate calls properly.
```

Comment:
Create an issue to address this TODO before you merge this PR. After, I would revise this to say:
`TODO(@loic-sharma): #<PR_NUMBER>`

Reply:
Created this issue: https://github.com/flutter/flutter/issues/108404